### DISCUSSION TITLE
Why is the soft update rate of target networks (α) set to 0.1?

### DISCUSSION THREAD

**Discussion by Koyonomi** [Cryolite/kanachan#59] [Q&A] [answered]
In other open-source projects, this value is usually set to a number within the range of [0.001, 0.01], so 0.1 seems significantly larger. In Kanachan, it is set to 0.1. Is it because smaller values make training very difficult, or is it solely for the purpose of accelerating convergence?

**Comment by Cryolite** [comment, marked as answer]
Your are absolutely correct. For instance, in the original papers on CQL and IQL, the experimental settings for the target network update ratio (Polyak averaging coefficient) used 0.005, while the default value in kanachan is 0.1, which is significantly larger. This is simply an error without any particular intent, and it seems advisable to change the default value. This change will be promptly reflected in the `develop` branch.

**Reply by Koyonomi** [reply]
Okay, I thought this setting was set to 0.1 for some specific motive, but it seems that's not the case. Thanks for your help!
